src/confluent_kafka/src/Consumer.c

```diff
@@ -1281,9 +1419,9 @@ static PyMethodDef Consumer_methods[] = {
 	  "\n"
 	  "  Actions performed:\n"
 	  "\n"
-	  "  - Stops consuming\n"
-	  "  - Commits offsets - except if the consumer property 'enable.auto.commit' is set to False\n"
-	  "  - Leave consumer group\n"
+	  "  - Stop consuming.\n"
```

Comment:
"Actions performed (by this method)" would need this list to be in third-person form, thus "Stops", "Commits", "Leaves", etc.